packages/test/end-to-end-tests/package.json

```diff
@@ -25,12 +24,15 @@
     "lint": "npm run eslint",
     "lint:fix": "npm run eslint:fix",
     "start:tinylicious:test": "tinylicious > tinylicious.log 2>&1",
-    "test": "start-server-and-test start:tinylicious:test 3000 test:mocha",
```

Comment:
This looks like this because the root package.json starts the test server as part of the `test:mocha` script, so unless we also did it here we can't run tests directly from inside this dir, but we also can't start the test server twice.  If we expect this is the only package to ever start this test folder we could just move it here, but otherwise I think the `start-server-and-test` part will need to run under a script that doesn't overlap with the lerna'd root script

Reply:
i don't love the design, but fixed for now